apisix/plugins/jwt-auth.lua

```diff
@@ -357,8 +361,31 @@ local function algorithm_handler(consumer, method_only)
     end
 end
 
+local function set_our_cookie(name, val)
+    core.response.add_header("Set-Cookie", name .. "=" .. val)
```

Comment:
This function doesn't hide the cookie from the upstream, right?

Reply:
Be glad to hear your advice. The first time I refer to the [cas-auth.lua](https://github.com/apache/apisix/blob/master/apisix/plugins/cas-auth.lua#:~:text=local%20function%20set_our_cookie(name%2C%20val)) plugin(`set_our_cookie()`) to update the cookie as deleted, and then I refer to the operation of the `lua-resty-cookie` [library](https://github.com/cloudflare/lua-resty-cookie). 
Finally I realized that it is actually calling the same handling about header's `Set-Cookie`. If you have a better operation suggestion, please let me know.